ui/src/utils/removeExtension.js

```diff
@@ -0,0 +1,2 @@
+// Remove extension from filename
+module.exports = filename => filename.replace(/\.[^/.]+$/, '');
```

Comment:
I wonder if something like `filename.split('.').pop()` would work instead of this module?